examples/src/main/java/io/dapr/examples/actors/http/README.md

```diff
@@ -108,6 +108,30 @@ cd to [repo-root]
 dapr run --app-id demoactorservice --app-port 3000 --port 3005 -- mvn exec:java -pl=examples -D exec.mainClass=io.dapr.examples.actors.http.DemoActorService -D exec.args="-p 3000"
 ```
 
+### Debugging the Demo actor service
+
+If you want to debug the `Subscriber`, you have to make sure to provide the port as an argument.
```

Comment:
Subscriber is a confusing name here, it should say actor service.

Reply:
Done